docs/src/prototypes/chatMessages/ControlMessages/index.tsx

```diff
@@ -0,0 +1,198 @@
+import * as React from 'react'
+import {
+  Avatar,
+  Tree,
+  ChatItemProps,
+  ShorthandCollection,
+  Chat,
+  Divider,
+  Ref,
+} from '@stardust-ui/react'
+import * as keyboardKey from 'keyboard-key'
+import overridenTreeItemBehavior from './overridenTreeItemBehavior'
+import overridenTreeBehavior from './overridenTreeBehavior'
+import overridenTreeTitleBehavior from 'docs/src/prototypes/chatMessages/ControlMessages/overridenTreeTitleBehavior'
+
+const janeAvatar = {
+  image: 'public/images/avatar/small/ade.jpg',
+  status: {
+    color: 'green',
+    icon: 'check',
+  },
+}
+
+const treeItems = [
```

Comment:
These two items are almost identical. How about extracting the code into a function?

Reply:
will refactor it once we decide on the final implementation